guides/common/modules/proc_salt-creating-a-host-group-with-salt.adoc

```diff
@@ -0,0 +1,19 @@
+[id="salt_guide_creating_a_host_group_with_salt_{context}"]
+= Creating a Host Group With Salt
+
+To bundle provisioning and configuration settings, you can create a host group with Salt enabled for managed hosts.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure > Host Groups*.
+. Click *Create Host Group*.
+.. On the *Host Group* tab, select a *Salt Environment* and a *Salt Master*.
+.. On the *Salt States* tab, assign *Salt States* to your host group.
```

Comment:
```suggestion
.. Click the *Salt States* tab, and assign *Salt States* to your host group.
```
I'm assuming the Host Group tab is open by default? If not, maybe conform it to this style also.

Reply:
Host Group tab is open by default.

Applied to all three "on the X tab" sentences.